src/convertToObject.js

```diff
@@ -13,6 +13,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const sourceStringToArray = sourceString
+    .split('\n')
```

Comment:
It is a bad idea to split by newline here. Test don't cover it here, but styles can all be written in one line.
More sustainable solution is to split by `;`